HISTORY.rst

```diff
@@ -5,6 +5,10 @@ History
 
 * Added the following new values to the ``/payment/processor`` validation:
   ``ebs``, ``hipay``, and ``lemon_way``.
+* Added the following new values to the ``/device`` validation:
+  ``session_age`` and ``session_id``.
+* Added the following new values to the ``/email`` validation:
```

Comment:
This seems wrong. We added the attribute to the `Email` response model or, alternatively, we added support for the `/email/first_seen` output. We didn't do anything related to validation for this output.